tensorflow/core/kernels/mkl_tfconv_op.h

```diff
@@ -57,6 +61,71 @@ class MklToTfOp : public OpKernel {
     VLOG(1) << "MKLToTFConversion complete successfully.";
   }
 
+#ifdef INTEL_MKL_DNN
+  static void ConvertMklToTf(OpKernel* op_kernel, OpKernelContext* context,
+                             string data_format_str, DataType op_data_type,
+                             bool has_avx512f, uint input_number) {
+    try {
```

Comment:
Google style-guide does not allow C++ exceptions.
https://google.github.io/styleguide/cppguide.html#Exceptions

Reply:
@caisq can you start the test here? I think @andydavis1 is ok with the changes.

Reply:
Note for posterity,
In opensource tensorflow, we enable exceptions only for MKL.
Rest of the code has exceptions disabled.